app/models/results.js

```diff
@@ -13,12 +12,13 @@ const _ = require('lodash');
 const FileSchema = require('./extends/file');
 const tools = require('../tools');
 
+// Model variables
 const Schema = mongoose.Schema;
-const checksum = tools.checksum;
 const filedb = tools.filedb;
 const fileProvider = filedb.provider;
 const Build = mongoose.model('Build');
 
+// @Todo justify why file schema is extended here instead of adding to root model
```

Comment:
file - (related to individual result) could related to individual Resource (object) and here we are extending more details to FileSchema which allows to link `Result.file` to `Resource` document.. In this case this is implemented little wrong way because it extends original FileSchema instead of creating new one.. Maybe - maybe these could/should be moved to original FileSchema ...

Reply:
or re-think how it should be..